react_components/index.js

```diff
@@ -5,6 +5,16 @@ import '../css/fontello.css';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import CardComponent from './Card';
+import {addLocaleData, IntlProvider} from 'react-intl';
```

Comment:
without importing that, how will I be able to add the specific locale data pertaining to specific languages ?